src/go.mod

```diff
@@ -36,6 +36,7 @@ require (
 	golang.org/x/sys v0.20.0
 	golang.org/x/term v0.18.0
 	google.golang.org/protobuf v1.33.0
+	gopkg.in/yaml.v2 v2.2.8
```

Comment:
look under this line: we already use yaml.v3, so we shouldn't also need to use v2 of the same library